setup.cfg

```diff
@@ -33,15 +32,22 @@ packages = find:
 include_package_data = True
 
 [options.extras_require]
+acq = 
+    bioread >=1.0.5
```

Comment:
This was `bioread>=1.0.4` in `.travis.yml`. We should be testing with the same version of bioread. Are you sure this line should not be `bioread>=1.0.4`?

Reply:
Ah, good catch. I have to update the bioread version in Travis as well then.

Since yesterday one of the points of this PR became updating the minimal bioread version requirement (I made a PR to it that was released in 1.0.5).
Sorry for not having updated the general description!